packages/dd-trace/src/config.js

```diff
@@ -134,6 +134,11 @@ class Config {
       process.env.DD_APPSEC_RULES,
       path.join(__dirname, 'appsec', 'recommended.json')
     )
+    const DD_APPSEC_TRACE_RATE_LIMIT = coalesce(
+      appsec.rateLimit,
+      process.env.DD_APPSEC_TRACE_RATE_LIMIT,
+      100
```

Comment:
Should this default to the existing rate limit instead? The reasoning is that if a user wants to allow for example 1000 traces per second, there is no reason to limit AppSec to 100 as that would be extremely unexpected. Basically, AppSec should always keep _more_ traces and not _less_ traces.

Reply:
@vdeturckheim wdyt ?

Reply:
I've asked Raphael, and he confirmed that it already works like this, because the two rate limiters are additive. Even if AppSec decides to stop sampling, the tracer can still decide to sample the trace. So if the tracer rate limit is superior to AppSec one's, the tracer rate limit will automaticall take over AppSec's one.